src/e-init.js

```diff
@@ -77,7 +79,7 @@ function runGClientConfig(config) {
     '--name',
     'src/electron',
     '--unmanaged',
-    'https://github.com/electron/electron',
+    config.remotes.electron.origin,
```

Comment:
Does this change still work on the default repo when `useHttps` is not set, i.e. when origin is `git@github.com:electron/electron.git`?

It looks like the `https://` being replaced here was here since the initial commit, so I can't tell from the comment history whether its use was intentional or if it snuck in by accident.